vms/shift/tests/test_manageVolunteerShift.py

```diff
@@ -480,6 +484,42 @@ def test_cancel_assigned_shift(self):
         self.assertEqual(slots_remaining_before_assignment,
                          slots_after_cancellation)
 
+    def test_shift_edit_request(self):
+        volunteer_1 = create_volunteer_with_details(self.volunteer_1)
+
+        shift = ['09:00', '15:00', '1']
+        shift_1 = self.create_shift(shift)
+        start = datetime.time(hour=10, minute=0)
+        end = datetime.time(hour=14, minute=0)
+        logged_shift = log_hours_with_details(volunteer_1, shift_1, start, end)
+        start_time = datetime.time(hour=9, minute=30)
+        end_time = datetime.time(hour=14, minute=0)
+        edit_request = create_edit_request_with_details(start_time, end_time, logged_shift)
+        shift_id = shift_1.id
+        volunteer_id = volunteer_1.id
+        edit_request_id = edit_request.id
+        response = self.client.get(reverse('shift:edit_request_manager', args=[shift_id, volunteer_id, edit_request_id]))
+        self.assertEqual(response.status_code, 302)
+
+    def test_edit_request_email_volunteer(self):
+        volunteer_1 = create_volunteer_with_details(self.volunteer_1)
+
+        shift = ['09:00', '15:00', '1']
+        shift_1 = self.create_shift(shift)
+        start = datetime.time(hour=10, minute=0)
+        end = datetime.time(hour=14, minute=0)
+        logged_shift = log_hours_with_details(volunteer_1, shift_1, start, end)
+        start_time = datetime.time(hour=9, minute=30)
+        end_time = datetime.time(hour=14, minute=0)
+        edit_request = create_edit_request_with_details(start_time, end_time, logged_shift)
+        vol_email = volunteer_1.email
+        mail.outbox = []
+        mail.send_mail("Log Hours Edited", "message", "messanger@localhost.com", [vol_email])
```

Comment:
Is this assuming Admin approved the request?

Reply:
There is no such thing like approval or disapproval, if the admin edits his hours on his request, he will get an email notifying the same.

Reply:
Okay